daemon/algod/api/server/v2/handlers.go

```diff
@@ -957,7 +970,8 @@ type PreEncodedSimulateRequestTransactionGroup struct {
 
 // PreEncodedSimulateRequest mirrors model.SimulateRequest
 type PreEncodedSimulateRequest struct {
-	TxnGroups []PreEncodedSimulateRequestTransactionGroup `codec:"txn-groups"`
+	TxnGroups     []PreEncodedSimulateRequestTransactionGroup `codec:"txn-groups"`
+	LiftLogLimits *bool                                       `codec:"lift-log-limits,omitempty"`
```

Comment:
Since we want to default to false anyway, I think you can get away with declaring this as `bool` instead of a pointer. That'd save some dereferencing work

Reply:
I guess eventually I will go with https://github.com/algorand/go-algorand/pull/5247#discussion_r1173116213, which means this field will not need to be reported?

Reply:
see [refactor Simulate interface, remove unnecessary codec struct, remove …](https://github.com/algorand/go-algorand/pull/5247/commits/61d8b7e19f08848a812d01c8933950740c97b60d)

Reply:
oh snap, understood it wrongly again. see [preEncodedSimReq LiftLogLimits ptr -> bool](https://github.com/algorand/go-algorand/pull/5247/commits/b778873becdf523c55f9f176d43160ea15647b4b)